src/Saml2/Utils.php

```diff
@@ -411,15 +411,15 @@ public static function post($url, array $parameters = array(), $stay = false)
         $fields = array();
         foreach ($post as $name => $value) {
             $fields[] = sprintf(
-                '<input type="hidden" name="%s" value=%s"/>',
+                '<input type="hidden" name="%s" value="%s"/>',
                 htmlspecialchars($name, ENT_QUOTES, 'utf-8'),
                 htmlspecialchars($value, ENT_QUOTES, 'utf-8')
             );
         }
 
         printf('
             <!DOCTYPE html>
-            <html><head><meta charset="utf-8"/>Redirect<title></title></head>
+            <html><head><meta charset="utf-8"/>Redirecting...<title></title></head>
             <body><form id="form" action="%s" method="post" enctype="application/x-www-form-urlencoded">%s
             <noscript><button type="submit">Click to continue</button></noscript></form>
             <script>document.getElementById("form").submit();</script></body></html>', 
```

Comment:
We might need to create something more visually appealing because the user will see the blank page for a moment… However, this can be handled by the user of the library (`post()` can return the URL and contents of the input fields)